src/AsyncResult.ts

```diff
@@ -223,7 +224,7 @@ function tryCatch<A, E = unknown>(
 function tryCatch<A, E = unknown>(
     mightThrow: Async<A>,
     onThrow?: (err: unknown) => E
-): AsyncResult<A, any> {
+): AsyncResult<A, unknown> {
```

Comment:
Is this legit / safe?

Reply:
If it compiles and the tests run, probably? Implementation signatures of overloaded functions are not externally visible, though, so I'm not sure whether it has any effect on calling code